examples/sampling_examples/vsr-token-holder-history.sql

```diff
@@ -0,0 +1,22 @@
+WITH
+  VSRTokenInformation AS (
+    SELECT
+      tx_signature AS signature,
+      update_authority
+    FROM
+      `solana-bq.test_usa.Tokens`
+  )
+SELECT DISTINCT
+  block_timestamp,
+  ptb.owner AS holder,
+FROM
+  `solana-bq.test_usa.Transactions` AS Transactions
```

Comment:
Please change this table name to use the new dataset name: `solana-data-sandbox.crypto_solana_mainnet_us.Transactions`

Reply:
@julian-bcw I do not have permission/access for this - should I ask Tornike for this?